Fix modification contraintes champ

dlevasseuraltitude opened the pull request:
Modification des contraintes sur les champs pour mise en conformité par rapport à la doc.

dlevasseuraltitude committed:
Update emutation.wsdl

Version finale AI

dlevasseuraltitude committed:
Merge remote-tracking branch 'upstream/master' into FIX-Modification-Contraintes-Champ

dlevasseuraltitude committed:
FIX | Mise en conformité doc

Mise à jour des champs pour mise en conformité par rapport à la doc

aminvielledebat commented on emutation.wsdl:
```diff
@@ -359,8 +359,8 @@
                 <xsd:sequence>
-                    <xsd:element name="Nom" type="xsd:string"/>
-                    <xsd:element minOccurs="0" name="Identifiant" type="xsd:string"/>
+                    <xsd:element name="Nom" type="xsd:string" minOccurs="0" />
```
Effectivement, ça me paraît plus judicieux.
Si c'était comme ça dans les specs et que c'est passé à travers, désolé.

aminvielledebat commented:
@dlevasseuraltitude tu as la spec mise à jour ? Parce que la version que j'ai possède encore les anciennes longueurs. Cela a été décidé en GT récemment ?

Autre chose, est ce que tous les OIs sont OK pour basculer directement dessus ? Sinon il faudra que les OCs s'adaptent (pas top).

NicolasD-Altitude commented:
@aminvielledebat  je t'ai envoyé la spec validée par le GT Accès. Et effectivement tous les OI sont ok.

NicolasD-Altitude commented:
Si tout le monde est OK on merge.

aminvielledebat reviewed (dismissed):
Par contre, il faudra faire passer le job travis sur java (étonnant que ça saute).
Une idée @jlsaxione ?

NicolasD-Altitude commented:
Attendons avant de valider, il semblerait que d'autres modifs interviennent.

aminvielledebat commented:
@NicolasD-Altitude @dlevasseuraltitude petit up pour la mise à jour des namespaces.

Pour les autres @jlsaxione, @olaissac-bt, @WMaraux, @mamraoui, une fois fait on valide pour géné ?
Attention cela veut dire que cette version devient la référence. Je ne sais pas où vous en êtes, mais on n'en déploiera qu'une de notre côté.

jlsaxione commented:
Petite question de mise en point. Si on modifie, cela invalide la version précédente. Etes vous en accord ?

aminvielledebat commented:
C'était le sens de ma remarque @jlsaxione  ;-)

jlsaxione commented:
juste pour être sur :).

NicolasD-Altitude commented:
J'attends un retour de Corinne Gérard car il semble que d'autres modifications ont été demandées pour cette 1.2.

dlevasseuraltitude committed:
FIX | Mise en cohérence documentation

Mise en cohérence des champs suivants selon la documentation:
-	ConnecteurPriseCouleur est précisé : alphanumérique(256)
-	Les champs CoordonneesX et CoordonnesY passent d'alphanumérique à numérique
-	Les champs CoordonneesX et CoordonnesY passent en obligatoire (au même titre que TypeProjection)
-	La taille des 5 champs (NomModulePM, PositionModulePM, ReferenceCableModulePM, InfoTubeModulePM, InfoFibreModulePM) est précisée : alphanumérique(256)

NicolasD-Altitude commented:
Bonjour, 
Peut-on valider rapidement, j'ai pris l'engagement au dernier GT de fournir le WSDL asap.

aminvielledebat commented:
Hello @NicolasD-Altitude, aucun problème pour moi pour valider, modulo la remarque sur les namespaces.

Aujourd'hui, partout dans le wsdl, on a ```http://interop-fibre.fr/Emutation/1.0/``` et pas ```http://interop-fibre.fr/Emutation/1.2/``` (si on est bien d'accord que cette version correspond à la 1.2 interop).

Je l'avais laissé passer sur les précédentes PRs, mais il faut que l'on soit rigoureux là dessus si on ne veut pas avoir des soucis sur nos types.

Dernière bricole, je vois que tu continue à version les modifs dans le ```README```. Dans l'absolu, c'est le tag de la version globale qui porte le diff, et il faudrait que les versions du ```README``` soient cohérentes avec celles du webservice (si tout le monde est d'accord avec ça bien sûr :-))

dlevasseuraltitude committed:
FIX | Correction namespace et readme

aminvielledebat reviewed (approved):
C'est bon pour moi.

Le souci sur travis n'a rien à voir. Il faudra faire une PR à côté pour remettre ça d'équerre.

Merci @NicolasD-Altitude @dlevasseuraltitude 

Merci
